app/gamelib/src/plot.rs

```diff
@@ -0,0 +1,107 @@
+use alloc::format;
+use alloc::vec::Vec;
+use alloc::vec;
+use alloc::string::String;
+use crate::alloc::string::ToString;
+use hashbrown::HashMap;
+use ogc_rs::println;
```

Comment:
This is why your unit tests are failing: you use wii specific imports. To fix this, do what we did in lib.rs:

```
#[cfg(feature = "wii")]
pub use ogc_rs::{print, println};
#[cfg(not(feature = "wii"))]
pub use std::{print, println};
```

You can do that for any wii-specific thing (check all your files here). The unit tests are compiled for PC.